### PR TITLE
2.0.8 widget rampage

### PR DESCRIPTION
**Pull request by thescubageek**
Converts the following widget's edit.html to the new WidgetEditorFilter liquid:

* Analytics
* City State Zip Search
* Call Outs
* Corporate Locations Map
* Coupon
* CSS Override
* Directions
* Directory
* Featured Properties
* Footer Info

### CODE REVIEW

**Comment by thescubageek** [comment]
This has been merged into 2.0.8-merge: https://github.com/g5search/g5-content-management-system/pull/740

**Comment by dreuss** [comment]
The refactor on all the edit pages looks consistent, and great reduction in repetitive code.  I might have @mmitchellg5 look over some of the ruby codes just to have a second set of eyes, but looks good. :+1:

File: app/filters/widget_editor_filter.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def edit_integer_field(setting, label="", placeholder="", method=:value, klass="
     ).html_safe
   end
 
-  def edit_textarea_field(setting, label="", placeholder="", method=:value, klass="")
+  def edit_textarea_field(setting, label="", placeholder="", method=:value, klass="", cols=40, rows=3)
```
Comment (variousred) [inline app/filters/widget_editor_filter.rb]: with this many settings, i might accept an options hash

Comment (thescubageek) [inline app/filters/widget_editor_filter.rb]: Hashes are really cumbersome to pass via liquid.... you have to do some ugly formatting and then use JSON.parse on the other end to make it work (see the edit_select_field etc)

Comment (variousred) [inline app/filters/widget_editor_filter.rb]: ah, gotcha

**Pull request closed without merging**